tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -377,13 +386,13 @@ spec:
 			})
 
 			stdOut := helper.CmdShouldPass("odo", "service", "list")
-			helper.MatchAllInOutput(stdOut, []string{"example", "EtcdCluster"})
+			helper.MatchAllInOutput(stdOut, []string{"etcdcluster", "EtcdCluster"})
```

Comment:
In line 366, you are creating the service with a default name. On main branch, it is created with `example` and in your branch with `etcdcluster`.

Is it intentional?

Reply:
Oh, yes it is: 
```
// if the service name is blank then we set it to custom resource name
```

Reply:
yeah, we are doing this change to make sure both approaches ( param and non-paramed ) would provide same default name